:sparkles: Issue 2 - Add Docker

Moosgloeckchen opened the pull request:
# Add Dockerization

* add docker-compose
* add docker adr
* add documentation for quick start

Fixes #11 und #12 

! Please test the dockerization via the documentation. Hope it works like a charm but if not let me know :)

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] This change requires a documentation update

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings

MaxHam commented on docker-compose.yml:
```diff
@@ -0,0 +1,10 @@
+version: "3.8"
```
Why do you use a docker-compose.yml ? Since you only have one container it may not be necessary, but I think it does not pose to be a problem.

Moosgloeckchen replied:
Good point! i figured if i go through the process of getting to know docker i might as well learn how it works also with multiple containers. Also that way i am prepared in case i have to add a container. :)

MaxHam reviewed (approved):
I cloned the repo, built the image and everything worked fine right away. 
I like the implementation of a detail view.
I have one question regarding the authentication: How does the `.htaccess` file work in your project?

All in all the changes of this PR are in a great shape

Moosgloeckchen commented:
> I cloned the repo, built the image and everything worked fine right away. I like the implementation of a detail view. I have one question regarding the authentication: How does the `.htaccess` file work in your project?
> 
> All in all the changes of this PR are in a great shape

Great, thank you! Regarding your question: It does not. I added it for deployment, but then decided on using github pages, so it does not work. I added a cleanup issue  for later (#20 ) thanks for raising. And for the review!
